### PR TITLE
Creat Fawo011

### PR DESCRIPTION
**Pull request by Fawo011**
I have create my own file.

### CODE REVIEW

**Comment by irvingskr** [comment]
Please sync-fork from the your source repository, your workflow check file have been out-of-date

**Comment by Fawo011** [comment]
> 

后边那句不是很明白

**Comment by irvingskr** [comment]
> > 
> 
> 后边那句不是很明白，check file是啥


Just sync-fork, it is just a file for me to check your work conveniently.

**Comment by Fawo011** [comment]
> > > 
> > 
> > 
> > 后边那句不是很明白，check file是啥
> 
> Just sync-fork, it is just a file for me to check your work conveniently.

i have done can u see it?

File: 2022-finalproject/Fawo011/solution.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+1. 使用git reset 不保留记录
+2. 保留记录使用git revert
+
+## Last question
+
+- 先创建一个分支testing
+- ![image](./7.png)
+- 进入master后可使用git merge testing 合并
+- ![image](./8.png)
+- 另一种方法，想了很久，只能想到用git cherry-pick命令了合并testing上的某个提交
```
Comment (Fawo011) [inline 2022-finalproject/Fawo011/solution.md:40]: - 这种方法建议创建一种新的分支合并再提交后push新的分支，此PR就是用这个方法提交

**Review by Fawo011 — COMMENTED** [review]
Having found nothing yet.

**Comment by irvingskr** [comment]
> Having found nothing yet.

我的意思是让你更新一下你的仓库，你在非master分支下pull request我的检查脚本跑不起来，我写了个新的脚本，你更新下仓库就行。

**Comment by Fawo011** [comment]
> 

哦还以为这是学长所说的叫我review一下

**Comment by Fawo011** [comment]
> > Having found nothing yet.
> 
> 我的意思是让你更新一下你的仓库，你在非master分支下pull request我的检查脚本跑不起来，我写了个新的脚本，你更新下仓库就行。

学长我review了怎么还显示Review required

**Comment by irvingskr** [comment]
> > > Having found nothing yet.
> > 
> > 
> > 我的意思是让你更新一下你的仓库，你在非master分支下pull request我的检查脚本跑不起来，我写了个新的脚本，你更新下仓库就行。
> 
> 学长我review了怎么还显示Review required

Review is my job...

File: 2022-finalproject/Fawo011/solution.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+![image](./2.png)
+
+- 窝草有错误！
+- 这时候运行git reset test.txt
+![image](./3.png)
+- 直接就是将他移出暂存区
+**抑或是git reset --hard**
+
+### 总结
+1. 使用git reset
```
Comment (irvingskr) [inline 2022-finalproject/Fawo011/solution.md:16]: Really?

**Review by irvingskr — APPROVED** [review]

**Pull request merged**